src/app/components/edit-data-set/edit-data-set.component.html

```diff
@@ -0,0 +1,73 @@
+<div class="container-fluid">
+  <div class="container">
+    <div class="row">
+      <div class="col">
+        <div class="display-1 h1-black">
+          <h3 class="h1-style">Parameters of {{dataSetTitle}}</h3>
+        </div>
+      </div>
+      <div class="col-sm">
+        <div class="container">
+          <div class="row">
+            <div class="col-sm">
+              <input #myInput class="form-control mr-sm-2 input-long" type="text" placeholder="Search" (keyup.enter)="doSearch(myInput.value)">
+              <button class="options ml-1 button-list" (click)="doSearch(myInput.value)">
+                Search
+              </button>
+            </div>
+            <div class="col-sm">
+            </div>
+            <div class="col-sm">
+              <button [routerLink]="[backToDataSet()]" class="options ml-1 button-list" color="primary">Go to data sets</button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-sm">
+        <table class="table table-striped table-config" *ngIf="isFound">
+          <thead class="thead">
+          <tr>
```

Comment:
I`m not sure, but I think that using /tr is deprecated. As I know, now tables are made by simply /dev blocks.